src/sdk/pynni/nni/compressors/example/main_tf_pruner.py

```diff
@@ -0,0 +1,115 @@
+from nni.compressors.tfCompressor import AGPruner
```

Comment:
suggest change 'main' in file names to 'mnist'

Reply:
it maybe confusing with the class Mnist